.github/CODEOWNERS

```diff
@@ -9,20 +9,21 @@
 /sdk/ @joshfree @jonathangiles
 
 # Service teams
-/sdk/appconfiguration/       @alzimmermsft @conniey @sima-zhu
-/sdk/batch/                  @xingwu1 @bgklein @matthchr
-/sdk/core/                   @alzimmermsft @jianghaolu @srnagar @hemanttanwar
-/sdk/cosmos/                 @moderakh @christopheranderson @kushagraThapar
-/sdk/eventhubs/              @conniey @srnagar @mssfang
-/sdk/identity/               @jianghaolu @g2vinay @hemanttanwar
-/sdk/keyvault/               @g2vinay @samvaity
-/sdk/servicebus/             @yvgopal @nemakam
-/sdk/tracing/                @samvaity @alzimmermsft
+/sdk/appconfiguration/       				@alzimmermsft @conniey @sima-zhu
```

Comment:
It looks like this line will have a merge conflict possibly due to a recent change in master

Reply:
I have resolved the conflicts.